.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: Build
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    types: [opened, reopened, synchronize]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 13.x, 14.x]
```

Comment:
For the record: this also updates the tested Node versions from 6, 8 to 10, 12, 13, 14.